internal/core/ports/claims_repository.go

```diff
@@ -27,4 +27,5 @@ type ClaimsRepository interface {
 	GetAuthClaimsForPublishing(ctx context.Context, conn db.Querier, identifier *core.DID, publishingState string, schemaHash string) ([]*domain.Claim, error)
 	UpdateClaimMTP(ctx context.Context, conn db.Querier, claim *domain.Claim) (int64, error)
 	Delete(ctx context.Context, conn db.Querier, id uuid.UUID) error
+	GetClaimsIssuedForUser(ctx context.Context, conn db.Querier, identifier *core.DID, userDID *core.DID, linkID uuid.UUID) ([]*domain.Claim, error)
```

Comment:
```suggestion
	GetClaimsIssuedForUser(ctx context.Context, conn db.Querier, identifier core.DID, userDID core.DID, linkID uuid.UUID) ([]*domain.Claim, error)
```
I think that identifier and userDID shouldn't be nullable, as these are mandatory parameters.